source/administration/admin-adviser-experience.rst

```diff
@@ -0,0 +1,68 @@
+Admin Adviser Experience (AAE)
+===============================
+
+.. note::
+  Mattermost AAE Message Codes were introduced in version 5.26. They are added retroactively to existing System Admin messages. 
+
+Administrator Adviser Experience ("AAE") is a set of notifications and workflows to guide System Admins through the proper activation and configuration of system capabilities as the needs of their user base evolves. 
+
+AAE includes: 
+
+- Bot notifications to System Admins
+- Notification alerts in the channel header (only visible to System Admins)
+- A coding system to look up more information on errors and warnings
+
+AAE Message Code System 
+-------------------------------
+
+Since Mattermost may be deployed in air-gapped networks where links to documentation aren’t always available, AAE uses human-readable message codes to uniquely identify errors, warnings and recommendations. These codes can be manually referenced in downloaded documentation, or through web search on a separate internet-connected device. 
+
+Example: Please check connection, Mattermost unreachable. If issue persists, ask administrator to check WebSocket port. AAE-000-0010
+
+The "AAE-010-0010" message code represents an entry within a directory of message codes that can be downloaded locally and used to troubleshoot common issues.
+
+AAE Message Code Syntax
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+AAE Message Codes consists of 12 characters, with the first four characters starting with "AAE-" followed by a three digit "Category & Subcategory" code, a hyphen, then a "Message ID" code. The format specifically is: 
+
+"AAE-" & [Category & Subcategory] & "-" & [Message ID]
+
+AAE Category & Subcategory 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+AAE Message Codes are categorized as follows:
+
+.. csv-table::
+    :header: "Category", "Subcategory", "Category & Subcategory Code"
+
+    "System Administration", "Common Issues", "010"
+
+The classification of AAE codes is an iterative work in progress. We will use the AAE-010-XXX0 name space for the first few hundred issues, then analyze, categorize and subcategorize the AAE Categories and Subcategories, migrate to new codes, and deprecate the original AAE-010-XXX0, with information preserved for legacy deployments. 
+
+AAE Message Code Registry 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following section lists out AAE codes with links to troubleshooting documentation.
+
+.. csv-table::
+    :header: "AAE Code", "Initial Release Version", "Message", "More Information"
+
+    "AAE-010-0010", "5.26", "Please check connection, Mattermost unreachable. If issue persists, ask administrator to check WebSocket port. AAE-000-0010", "Potential proxy configuration issue. See `troubleshooting guide <https://docs.mattermost.com/install/troubleshooting.html#please-check-connection-mattermost-unreachable-if-issue-persists-ask-administrator-to-check-websocket-port>`_ to diagnose network connection."
```

Comment:
Note to self - use `:doc:` reference instead of a hardcoded link to a doc section (which is vulnerable to breaking links)